### ISSUE TITLE
feat: other language support

### ISSUE THREAD

**Issue by sekwah41** [roldanjr/pomatez#222]
Integrate with a website like https://crowdin.com/ or https://localazy.com/ to allow community translations we can easily merge in. Though it would be great to add support. We should try to auto detect from the browser but also allow a setting to override. Falling back to english whenever we have the values missing.

Crowdin is the same website Mojang uses and is free for public projects. https://crowdin.com/project/minecraft

**Comment by sekwah41** [comment]
I will add more info once I have properly implemented this on another project of mine as an experiment.

**Comment by tcitr-chen** [comment]
Hello, is there any progress so far?

**Comment by sekwah41** [comment]
Sadly not, we've been focused on other features when we've had time to work on this.

**Comment by huangyinhaow** [comment]
+1

**Comment by roldanjr** [comment]
Yes sure @huangyinhaow, after we release the revamped version of this app, will try to support the internationalization feature so everyone can translate it into their own language if they wanted to :)

**Comment by Glandos** [comment]
As a developer AND translator, I would advise to go either with crowdin.com or [Weblate](https://hosted.weblate.org/).
Localazy has a bad workflow when translating, missing shortcuts, and you always need to jump from the keyboard to the mouse, which is not ideal when translating a lot of text.

**Comment by sekwah41** [comment]
will look into external tools but for now will just be code files
